unitOneAssessmentPractice.js

```diff
@@ -100,6 +203,17 @@ let assert = require('assert')
 // Write a function called containsOnlyNumbers that returns whether or not an array contains only numbers
 // Your function should use a higher-ordered function (e.g map, filter, reduce, every, sort) in its implementation
 // Hint: the isNaN() function will tell you whether something is not a number
+function containsOnlyNumbers(arr) {
+  return arr.every(el => {
```

Comment:
Good use of every. This can also be shortened as this: 
function containsOnlyNumbers(arr) {
  return arr.every(el => typeof el === "number")
}